src/otx/algo/detection/yolox.py

```diff
@@ -287,6 +141,10 @@ def _build_model(self, num_classes: int) -> SingleStageDetector:
             num_classes=num_classes,
             in_channels=96,
             feat_channels=96,
+            loss_cls=CrossEntropyLoss(use_sigmoid=True, reduction="sum", loss_weight=1.0),
+            loss_bbox=IoULoss(mode="square", eps=1e-16, reduction="sum", loss_weight=5.0),
+            loss_obj=CrossEntropyLoss(use_sigmoid=True, reduction="sum", loss_weight=1.0),
+            loss_l1=L1Loss(reduction="sum", loss_weight=1.0),
```

Comment:
This change doesn't make sense to me, so I was wondering if you could explain a bit more about why this change is necessary?

Reply:
I intended to explicitly show what loss functions are used, not entering that class.